package-lock.json

```diff
@@ -2,27 +2,445 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
-    "@babel/runtime": {
-      "version": "7.11.2",
-      "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.11.2.tgz",
-      "integrity": "sha512-TeWkU52so0mPtDcaCTxNBI/IHiz0pZgr8VEFqXFtZWpYD08ZB6FaSwVAS8MKRQAP3bYKiVjwysOJgMFY28o6Tw==",
+    "@babel/code-frame": {
```

Comment:
Sorry @jessica-lau, I actually just noticed that this is from the root folder. 

We probably want to avoid making changes to this one and also within the cypress service, unless new packages are added. Could you take a look and see if you can remove them? I could do it from my end if needed. Let me know